tests/RandomTest.php

```diff
@@ -380,7 +380,7 @@ public function testGetTitle()
 
         $m->load(2);
         $this->assertSame('Sue', $m->getTitle()); // loaded returns title_field value
-        $this->assertSame([1 => 'John', 2 => 'Sue'], $m->getTitles()); // all titles
+        $this->assertSame([2 => 'Sue'], $m->getTitles()); // traverse loaded
```

Comment:
I don't think this is correct. `getTitles()` should return names of all records taking applied conditions into account, but it's not related to is model loaded or not. It's like `$m->export([$m->title_field])`.